pkg/asyncexec/cmd.go

```diff
@@ -0,0 +1,103 @@
+package asyncexec
+
+import (
+	"bufio"
+	"os/exec"
+)
+
+// New instantiate a new Cmd object.
+func New(cmd *exec.Cmd, buff int) *Cmd {
+	return &Cmd{
+		cmd:      cmd,
+		stopChan: make(chan struct{}),
+		StdoutCh: make(chan []byte, buff),
+		StderrCh: make(chan []byte, buff),
+	}
+}
+
+// Cmd is a wrapper arround exec.Cmd. Mainly used to execute
+// command asynchronously with/or without output stream.
+type Cmd struct {
+	cmd *exec.Cmd
+
+	stopChan chan struct{}
+	StdoutCh chan []byte
+	StderrCh chan []byte
```

Comment:
These should be lowercase because they're exposed through getters?

Reply:
yes ++. Fixed